RasimKhusaenov/my_own_blog
# Add search by articles

### Summary

Added search by title and content of article using one search input.

### How it works

![image](https://user-images.githubusercontent.com/35406008/179572080-0d59791b-db45-442c-bc6c-3361f04a25d8.png)
![image](https://user-images.githubusercontent.com/35406008/179572148-a58301fd-50e9-426f-8ba6-57737cd34925.png)

### Test plan

List of steps to manually test introduced functionality:

- Go to https://my-own-blog-pr-72.herokuapp.com/
- Fill search form with some text from title or content of any article in site
- Make sure that only target articles displayed 

### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, ENV variables, services, scripts, etc.

### Checklist:


### References
- Resolves #38 Issue
